float64 a1
float64 a2

float64 b1
float64 b2


bool straight_path

float64 radius

bool right

float64 t_f

bool new 

bool pen_down
